python/ray/serve/tests/unit/test_autoscaling_policy.py

```diff
@@ -42,12 +49,22 @@ def test_bounds_checking(self):
             assert min_replicas <= desired_num_replicas <= max_replicas
 
     @pytest.mark.parametrize("target_requests", [0.5, 1.0, 1.5])
-    def test_scale_up(self, target_requests):
-        config = AutoscalingConfig(
-            min_replicas=0,
-            max_replicas=100,
-            target_num_ongoing_requests_per_replica=target_requests,
-        )
+    @pytest.mark.parametrize(
+        "use_target_ongoing_requests,use_target_num_ongoing_requests_per_replica",
+        [(True, True), (True, False), (False, True)],
```

Comment:
Same for this and all the ones below, would be great to test defaults, also maybe set `target_ongoing_requests` and `target_num_ongoing_requests_per_replica` to different values to check the behavior

Reply:
I think these tests are testing some concrete functionality so it requires setting target ongoing requests to a non-default value. For the tests that don't require this, I've already added testing for defaults. I've also added a test to `test_config.py` that sets them to different values.